cmd/crowdsec-cli/machines.go

```diff
@@ -21,6 +22,7 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
+	"golang.org/x/sys/unix"
```

Comment:
T

Reply:
?

Reply:
@rnagy : I reported @cuddytrippin for spam on other issues as well :/